app/views/account/bits.scala

```diff
@@ -31,11 +31,11 @@ object bits:
 
   def setting(name: Frag, body: Frag) = st.section(h2(name), body)
 
-  def radios[A](field: play.api.data.Field, options: Iterable[(A, String)], prefix: String = "ir") =
+  def radios[A](field: play.api.data.Field, options: Iterable[(A, String)], valueAliases: Option[Map[Option[String], Option[String]]] = None) =
     st.group(cls := "radio")(
       options.map { (key, value) =>
-        val id      = s"$prefix${field.id}_$key"
-        val checked = field.value has key.toString
+        val id      = s"ir${field.id}_$key"
+        val checked = (field.value has key.toString) || (~((~valueAliases).get(field.value)) has key.toString)
```

Comment:
not sure what's going on here but there has to be a better way

Reply:
(following the other comment above)

By splitting the pref into 2 bools, I am able to do like you said in your comment in the issue, to have 4 separate values, while being able to toggle one of those values (so zen in game being automatic remains independant from using zen in another place)